Makefile

```diff
@@ -56,6 +56,9 @@ spellcheck:
 	cat scripts/cspell/project-names.txt | grep -v '^#' | grep -v '^\s*$$' | tr ' ' '\n' > scripts/cspell/project-names-parsed.txt
 	cd scripts/cspell && ./spellcheck.sh
 
+fetch-blog-feed:
+	curl -s https://medium.com/feed/jaegertracing | yq -p xml -o json -P > data/medium.json
```

Comment:
Hugo can read XML file directly, there is no reason to convert to JSON (and JSON will introduce more escaping)

Reply:
I prefer not to require an extra dependency on yq